DigitalOffice/Services/ProjectService/Controllers/ProjectController.cs

```diff
@@ -15,6 +19,25 @@ public Project GetProjectInfoById([FromServices] IGetProjectInfoByIdCommand comm
             return command.Execute(projectId);
         }
 
+        [HttpGet("getProjectFileById")]
+        public async Task<object> GetProjectFileById(
+            [FromQuery] Guid id,
+            [FromServices] IRequestClient<IGetFileRequest> client)
+        {
+            var request = new
+            {
+                Id = id
+            };
+
+            var response = await client.GetResponse<IOperationResult<object>>(request);
+            if (!response.Message.IsSuccess)
+            {
+                throw new Exception();
+            }
```

Comment:
Must be exceptions processing for response message from broker. And this all logic must be move in other method, example: Sender, there is call only Execute method.